extensions/ql-vscode/test/data/lib/semmle/code/cpp/Namespace.qll

```diff
@@ -0,0 +1,237 @@
+/**
```

Comment:
Minor: I think a comment saying this is test data would also be a helpful reminder. The contents don't matter so much as the fact that it exercises various QL features.

Reply:
Fixed.